src/views/ContactView.vue

```diff
@@ -84,14 +91,24 @@ export default {
         email: "",
         topic: contactTopics[0],
         message: ""
-      }
+      },
+      sendState: sendStates.UNSENT
     };
   },
   computed: {
     ...mapGetters({
       isAuthenticated: "user/isAuthenticated"
     }),
-
+    sendLabel() {
+      switch (this.sendState) {
+        case sendStates.ERROR:
+          return "There was an error sending your feedback";
```

Comment:
I think the error message might be better suited above the send button instead of changing the button text

Reply:
Now Moved above the button